documentation/docs/features/summarization.md

```diff
@@ -0,0 +1,19 @@
+---
+sidebar_position: 5
+---
+
+# File Summarization
+
+You can use the `/summarize` command to get Khoj to generate context driven summaries of your documents.
```

Comment:
Can summarization also be triggered via natural language (e.g by just typing "summarize the file in one paragraph" or can it only be done by using the /summarize command?

Reply:
not currently.

Reply:
It can! I'd added this change in one of the fixes I'd made to your summarizer branch. I'll also update the docs.